meganav.yaml

```diff
@@ -429,6 +429,9 @@ products:
             - title: IoT Professional Services
               description: Bring IoT devices to market
               url: /core/services
+            - title: Install Ubuntu on NVIDIA Jetson
+              description:
```

Comment:
Sorry, gh didn't load this one with the initial review, but missing description shows up as "none"
![image](https://github.com/canonical/ubuntu.com/assets/17607612/0d007c47-a5c9-4fc1-aa88-cee47e13782a)

Reply:
Sorry, we are waiting on Felicia. I will add a note